generate/convert.go

```diff
@@ -262,6 +262,13 @@ func (g *generator) convertType(
 		// options work, recursing here isn't as connvenient.)
 		// Note this does []*T or [][]*T, not e.g. *[][]T.  See #16.
 		goTyp = &goPointerType{goTyp}
+	} else if !options.PointerIsFalse() && (options.GetPointer() || (!typ.NonNull && g.Config.Optional == "pointer_omitempty")) {
```

Comment:
I think you may actually need to do this in two cases, one for omitempty and one for pointer. Otherwise, `pointer: true` overrides the global config, which is a no-op for `pointer` but actually disables `omitempty`. I'll comment in the test below where that happens.